contractsV5/mockContracts/SafeERC20HelperV5.sol

```diff
@@ -0,0 +1,128 @@
+pragma solidity 0.5.9;
+
```

Comment:
what is the purpose of this contract?

Reply:
It's taken from the OpenZeppelin library for testing tokens that don't fully conform to the ERC20 standard.
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/mocks/SafeERC20Helper.sol

Think I removed some tests with regards to listing such tokens. Will add those tests back in.

Reply:
Do we use it in our deployed code?
if not, why do we need it here?